modules/@apostrophecms/launder/index.js

```diff
@@ -3,11 +3,11 @@
 // Apostrophe to sanitize user input.
 
 module.exports = {
-  init(self, options) {
+  init(self) {
     self.apos.launder = require('launder')({
       // A3 _ids may contain :-separated components
       idRegExp: /^[A-Za-z0-9_-]+(:[A-Za-z0-9_-]+)*$/,
-      ...options
+      ...self.options
```

Comment:
No, because launder doesn't do anything fancy or expensive with its options object like try to sanitize everything recursively and omitting the apos object would thus itself be overhead. In theory we could maintain logic here to pass only options to launder that are valid, but that would add to our maintenance burden, so I would say not today at least.